src/team_state.py

```diff
@@ -90,14 +142,41 @@ def from_config(cls, team_state: Dict[str, Any]):
             outs_for_inning,
             lineup,
             starting_pitcher,
-            stats,
+            stlats,
+            game_stats,
             blood,
+            cur_battter_pos,
         )
 
     def get_batter_stats_by_position(self, position: int) -> Tuple[str, Dict[str, float]]:
         batter = self.lineup[position]
         stats = self.stats[batter]
         return batter, stats
 
-    def get_player_stats_by_id(self, id: str) -> Dict[str, float]:
-        return self.stats[id]
+    def get_player_stats_by_id(self, player_id: str) -> Dict[str, float]:
+        return self.stats[player_id]
+
+    def next_batter(self) -> None:
+        if len(self.lineup) == self.cur_batter:
+            self.cur_batter_pos = 1
+        else:
+            self.cur_batter_pos += 1
+        self.cur_batter = self.lineup[self.cur_batter_pos]
+
+    def update_stat(self, player_id: str, stat_id: Stats, value: float) -> None:
+        self.game_stats[player_id][stat_id] += value
+
+    def get_defense_feature_vector(self) -> List[float]:
+        return []
```

Comment:
I assume this is a todo still? same with the next 3

Reply:
Yep.  Working on it.